### PR TITLE
Refactor sample skill to be more modular

### PR DESCRIPTION
**Pull request by selvp**
*Issue #, if available:*

*Description of changes:*
1. Refactor skill lambda to be more modular.  Introduced models/dao etc
2. Changed design to use single table instead of 2 tables. 
3. Updated ReadMe to reflect the end state.

### CODE REVIEW

File: sample_lambda/nodejs/index.js [hunk 1]
```diff
@@ -0,0 +1,393 @@
+/**
+ * Copyright <YEAR> Amazon.com, Inc. and its affiliates. All Rights Reserved.
```
Comment (srasthana) [inline sample_lambda/nodejs/index.js]: update <YEAR> with current year.

File: sample_lambda/nodejs/index.js [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+          resolve(skillResponse.newTimerResponse(healthMeasurementKeyObj.interfaceName, "Resume", {
+            currentNursingSide: payloadJson.nursingStartSide
+          }));
+        }
+      });
+  });
+}
+
+/*
+  Samples kill returns error response if an unknown directive and heath measurement type
```
Comment (srasthana) [inline sample_lambda/nodejs/index.js]: sample skill

File: README.md [hunk 3]
```diff
@@ -1,14 +1,57 @@
...
+       ```
+       cd <your workspace directory>
+       git clone https://github.com/alexa/alexa-baby-activity.git
+       cd alexa-baby-activity/sample_lambda/nodejs
+       npm install --prefix=./
+       zip -r alexa-baby-activity-sample.zip .
+       ```
+       3. Upload the zip file to lambda. Use sample_messages provided in this repository to create test events and test the lambda.
+
+   3. In Alexa developer console, Update lambda endpoint in  new skill to the lambda that was created by cloud formation template.
```
Comment (srasthana) [inline README.md]: You would also need to add Alexa skill kit trigger to the lambda. more details at https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-an-aws-lambda-function.html#configuring-the-alexa-skills-kit-trigger

File: sample_lambda/nodejs/index.js (hunk 1 above)
Comment (selvp) [reply sample_lambda/nodejs/index.js]: Changed to 2019

File: README.md (hunk 3 above)
Comment (selvp) [reply README.md]: Added a step to configure trigger lambda

File: sample_lambda/nodejs/index.js (hunk 2 above)
Comment (selvp) [reply sample_lambda/nodejs/index.js]: fixed

**Pull request merged**